medcat/utils/checkpoint.py

```diff
@@ -132,12 +174,72 @@ def _get_steps_and_count(file_path) -> Tuple[int, int]:
         file_name_parts = os.path.basename(file_path).split('-')
         return int(file_name_parts[1]), int(file_name_parts[2])
 
-    @staticmethod
-    def _load_metadata(dir_path: str) -> Optional[Dict]:
-        ckpt_file_paths = [os.path.abspath(os.path.join(dir_path, f)) for f in os.listdir(dir_path)]
-        metadata_file_paths = [f for f in ckpt_file_paths if os.path.isfile(f) and f.endswith("checkpoint-meta.json")]
-        if metadata_file_paths:
-            with open(metadata_file_paths[0]) as f:
-                return jsonpickle.decode(f.read())
-        else:
-            return None
+
+class CheckpointUT(Checkpoint):
```

Comment:
This and the supervised class are exactly the same, do not see a reason why we need two? How is one model checkpoint different from the other?

Reply:
They only defer in `DEFAULT_BASE_DIR`. This module will be further refactored after making checkpointing configurable in `config.py`.

Reply:
Ok, will that be done as part of this PR?